wren-base/src/test/java/io/wren/base/sqlrewrite/analyzer/TestDecisionPointAnalyzer.java

```diff
@@ -280,6 +305,28 @@ public void testRelation()
         else {
             throw new AssertionError("wrong type");
         }
+
+        statement = parseSql("SELECT * FROM (customer JOIN (SELECT 1 as custkey, 'xxx' as name) orders(custkey, name) ON customer.custkey = orders.custkey) join_relation");
```

Comment:
@cyyeh You can notice this case. The join criteria contains two expression. However, we will only get one in `exprSources`, see L324-L325.